uropa/visualization.py

```diff
@@ -26,29 +30,71 @@ def distribution_plot(table, var, kind):
     """
     pass
 
-
-def count_plot(table, var, kind):
+ef count_plot(table, var="feature", kind="pie", title=None, title_size=20, path=None, dpi=300.0, label_rot=45):
     """
     Count and plot the occurence of the selected categorical variable.
     Either shown as a pie chart or bar plot.
-
     Parameters
     ----------
     table : pd.DataFrame
         Pandas dataframe containing the data.
-    var : <datatype>, <default value>
-        <param description>
-    kind : <datatype>, <default value>
-        <param description>
-    TODO add more parameters
-
+    var : String, default="featire"
+        Value naming column along which to group peaks
+    kind : String, default="pie"
+        Value naming plot type (pie, bar)
+    title : String, default=None
+        Value for title of plot. If None plot has no title.
+    title_size: Integer, default=20
+        Value for size in points of figure title.
+    path : String, default=None
+        Value with path to save plot at. If None plot is not saved.

```

Comment:
Does this inlcude filename? Or just the path?

Reply:
It includes the filename and file ending as well. I added that to the description of the variable.